themebuilder-scss/src/data/bootstrap-metadata/bootstrap-metadata.ts

```diff
@@ -14,7 +14,7 @@ export default {
   'base-danger': '@brand-danger',
 
   'base-link-color': '@link-color',
-  'base-hover-color': '@table-bg-hover',
+  'base-hover-color': '@table-bg-accent',
 
```

Comment:
Is this right? Should we use table accent bg as hover?

Reply:
No...
I suggest replace on '@text-color', because Bootstrap4 inherit list: $table-hover-color -> $table-color -> $body-color